_components/list/guidance/when-to-consider-something-else.md

```diff
@@ -1 +1 @@
-- If you need to communicate long lists of narrative text.
+- **If you need to communicate long lists of narrative text.**
```

Comment:
It isn't very clear what this means to me. Can we remove or add clarity?

Reply:
Maybe this should simply say that you shouldn't use a list for narrative text or state that list items should be succinct and scannable or something to that effect.

Reply:
Going to leave it as is for now since it is out of scope. 
In the future, it might be helpful to point to https://www.plainlanguage.gov/guidelines/organize/use-lists/